com.unity.netcode.gameobjects/Runtime/NetworkVariable/Collections/NetworkDictionary.cs

```diff
@@ -444,6 +350,10 @@ public int LastModifiedTick
                 return NetworkTickSystem.NoTick;
             }
         }
+        public void Dispose()
```

Comment:
I need help understanding why I have to do this disposal here and not in a destructor, because this creates what is to me an unreasonable requirement that the user has to remember to explicitly dispose this instance

Reply:
Couldn't we do this automatically? All NetworkDictionary instances are managed by NetworkBehaviour right?

Reply:
I tried calling in `NetworkBehaviour::OnDestroy` the dispose method on all `NetworkVariables`, but this didn't work - the sentinel that triggers in the NativeList appears to check before the call is made.  I really don't like the alternative, which is where the user has to call stuff manually.   I have a question out in the Collections channel